pom.xml

```
@@ -5,7 +5,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.xspec</groupId>
   <artifactId>xspec</artifactId>
-  <version>1.4.0</version>
+  <version>1.4.0-SNAPSHOT</version>
```

We need to increment the version like we did in #568

```suggestion
  <version>1.5.0-SNAPSHOT</version>
```